irohad/ametsuchi/impl/storage_impl.cpp

```diff
@@ -272,20 +254,58 @@ namespace iroha {
       return wsv_command.insertPeer(peer);
     }
 
+    expected::Result<std::unique_ptr<MutableStorage>, std::string>
+    StorageImpl::createMutableStorageWithoutBlockStorage() {
```

Comment:
I think calling code should pass `BlockStorageStub` as a dependency to this method because we exclude code of `BlockStorageStub` from the storage and the code will be more maintainable - explicitly pass the dependencies.